pinot-query-runtime/src/test/java/org/apache/pinot/query/QueryTestSet.java

```diff
@@ -95,6 +95,10 @@ public Object[][] provideTestSql() {
             + " WHERE a.col1 IN ('foo') AND b.col2 NOT IN ('')"},
 
         // Range conditions with continuous and non-continuous range.
+        new Object[]{"SELECT a.col1, SUM(CASE WHEN a.col2 = 'foo' OR a.col2 = 'alice' THEN 1 ELSE 0 END) AS match_sum, "
```

Comment:
should we also add tests for v1 like `InTransformFunctionTest`?

Reply:
yeah I can add a test in v1 as well.

Reply:
Can we document this trick in the comments? :)

Reply:
added test

Reply:
Might want to add an explicit test in addition to the ones above (that test the functionality provided calcite rewrites) just to ensure this always works even if Calcite ever changes it's mind